runner/runner.go

```diff
@@ -231,60 +264,77 @@ func (r *Runner) run(ctx context.Context, checkMap map[string]*Check, list bool,
 		}
 	}
 
+	responseChannel := make(chan *Response, len(currentCheckList))
+	wg := &sync.WaitGroup{}
+	
 	// main loop to get the checks info.
 	for _, name := range currentCheckList {
-		resp := &Response{
-			name: name,
-		}
+		wg.Add(1)
+		xname := name
```

Comment:
If you do this you don't need to bind the variable as an argument to the closure on the next line - and conversely, if you pass `name` to the closure below on line 335, you don't need to do this local showing trick